DocumentedExamples/Cross-contour_transport.ipynb

```diff
@@ -0,0 +1,1480 @@
+       "        </details>\n",
+       "    </div>\n",
+       "</div>\n",
+       "            </details>\n",
+       "        \n",
+       "\n",
+       "    </div>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "<Client: 'tcp://127.0.0.1:42035' processes=4 threads=16, memory=44.92 GiB>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "client = Client(n_workers=4)\n",
+    "client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0fc07fcc-6e1d-40c0-be5b-c887266a20c7",
+   "metadata": {},
+   "source": [
+    "#### Choose database"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "142873eb-d834-40c2-aa5c-7b8f750f83d0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "session = cc.database.create_session('/g/data/ik11/databases/cosima_master.db')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de04e5ed-ee85-48ff-9628-abd3e6d8ea7b",
+   "metadata": {},
+   "source": [
+    "#### Choose experiment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "eed34fea-7cdd-4439-9668-3f543b9a4766",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "expt = '01deg_jra55v13_ryf9091'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "63761ef6-84b9-4c3b-9f99-9f90cb878e49",
+   "metadata": {},
+   "source": [
+    "#### Choose a latitude range so the contour fits in the range, but there is not too much extra space. Extra space slows down the computation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "900d12f3-548f-4854-8686-5dc9f3d037c7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lat_range = slice(-64.99,-47)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2ed43746-1772-4db8-be15-19accc9907ab",
+   "metadata": {},
+   "source": [
+    "#### We must make sure that this latitude range is so that the t-cells are always south and west of the u-cells.\n",
+    "\n",
+    "This is important because the meridional and zonal transports occur on different grids to each other. We can check this by loading the u-cell and t-cell coordinates."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "ab5a694c-b2a2-4f1d-9366-9a9facbf3fe5",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 10.3 s, sys: 3.33 s, total: 13.6 s\n",
+      "Wall time: 59.9 s\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "\n",
+    "yt_ocean = cc.querying.getvar(expt,'yt_ocean',session,n=1)\n",
+    "yt_ocean = yt_ocean.sel(yt_ocean=lat_range)\n",
+    "xt_ocean = cc.querying.getvar(expt,'xt_ocean',session,n=1)\n",
+    "yu_ocean = cc.querying.getvar(expt,'yu_ocean',session,n=1)\n",
+    "yu_ocean = yu_ocean.sel(yu_ocean=lat_range)\n",
+    "xu_ocean = cc.querying.getvar(expt,'xu_ocean',session,n=1)\n"
```

Comment:
We should write code as human-readable as we can. Although spaces after commas are not required by python, it makes the code much more readable by human eye. It's what we do with normal English, anyway. :)

```suggestion
    "yt_ocean = cc.querying.getvar(expt, 'yt_ocean', session, n=1)\n",
    "yt_ocean = yt_ocean.sel(yt_ocean = lat_range)\n",
    "xt_ocean = cc.querying.getvar(expt, 'xt_ocean', session, n=1)\n",
    "yu_ocean = cc.querying.getvar(expt, 'yu_ocean', session, n=1)\n",
    "yu_ocean = yu_ocean.sel(yu_ocean = lat_range)\n",
    "xu_ocean = cc.querying.getvar(expt, 'xu_ocean', session, n=1)\n"
```